docker/helpers/dnation_brand.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
There is already `docker/grafana/dnation/brand.sh` file which is used in the docker-compose manifest. 
Probably copy/paste error.